contrib/ci-builders/tekton/tekton-labs/tasks/zcash-build-test.yml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: zcash-build-test
+spec:    
+  params:
+    - name: JOBS
+      default: "4"
+      type: string
+    - name: BUILD_FOR_HOST
+      default: x86_64-pc-linux-gnu
+      type: string
+    - name: CONFIGURE_FLAGS
+      default: ""
+      type: string
+    - name: BUILD_CONTAINER
+      default: electriccoinco/zcashd-worker-ubuntu2004v2
+      type: string
+    - name: TEST_TARGET
+      default: ""
+      type: string
+  resources:
+    inputs:
+      - name: source
+        type: git
+  steps:
+    - name: build
+      image: $(params.BUILD_CONTAINER)
+      script: |
+        #!/bin/bash
+        set -o pipefail
+        cd workspace/source
+        echo $(git rev-parse HEAD)
+        echo $(git rev-parse --short HEAD)
+        
+        if [ "$(params.BUILD_FOR_HOST)" == "x86_64-apple-darwin11" ]
+        then
+            mkdir -p depends/SDKs
+            curl -fs https://ecc.mypinata.cloud/ipfs/QmeSwckvSCGL9SXGdEHoAyqXdzD7T9HYTwsC34Bj5EVDF5 \
+              -o depends/SDKs/Xcode-11.3.1-11C505-extracted-SDK-with-libcxx-headers.tar.gz
+            tar zxvf depends/SDKs/Xcode-11.3.1-11C505-extracted-SDK-with-libcxx-headers.tar.gz \
+              -C depends/SDKs
+        fi              
+
+        CONFIGURE_FLAGS=$(params.CONFIGURE_FLAGS) HOST=$(params.BUILD_FOR_HOST) ./zcutil/build.sh -j$(params.JOBS)
+
+    - name: test
+      image: $(params.BUILD_CONTAINER)
+      script: |
+        #!/bin/bash
+        set -o pipefail
+        python -m pip install pyblake2 pyflakes python-qpid-proton pyutil pyzmq requests simplejson
+        cp -a /home/.zcash-params $HOME/
+        echo $(git rev-parse HEAD)
+        echo $(git rev-parse --short HEAD)
+        HOST=x86_64-pc-linux-gnu /workspace/source/qa/zcash/full_test_suite.py $(params.TEST_TARGET)
```

Comment:
What is the purpose of setting `HOST` here and not having it set to `$(params.BUILD_FOR_HOST)`?

Reply:
This is to ensure we only build a target that can also run the full-test-suite.py. For now now that is only the Linux host, but later we can update this with ease.